lib/Entrypoint.js

```diff
@@ -2,37 +2,42 @@
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Tobias Koppers @sokra
 */
-function Entrypoint(name) {
-	this.name = name;
-	this.chunks = [];
-}
-module.exports = Entrypoint;
+"use strict";
 
-Entrypoint.prototype.unshiftChunk = function(chunk) {
-	this.chunks.unshift(chunk);
-	chunk.entrypoints.push(this);
-};
+class Entrypoint {
+	constructor(name) {
+		this.name = name;
+		this.chunks = [];
+	}
+
+	unshiftChunk(chunk) {
+		this.chunks.unshift(chunk);
+		chunk.entrypoints.push(this);
+	}
 
-Entrypoint.prototype.insertChunk = function(chunk, before) {
-	var idx = this.chunks.indexOf(before);
-	if(idx >= 0) {
-		this.chunks.splice(idx, 0, chunk);
-	} else {
-		throw new Error("before chunk not found");
+	insertChunk(chunk, before) {
+		const idx = this.chunks.indexOf(before);
+		if(idx >= 0) {
+			this.chunks.splice(idx, 0, chunk);
+		} else {
+			throw new Error("before chunk not found");
+		}
+		chunk.entrypoints.push(this);
 	}
-	chunk.entrypoints.push(this);
-};
 
-Entrypoint.prototype.getFiles = function() {
-	var files = [];
+	getFiles() {
+		let files = [];
 
-	for(var chunkIdx = 0; chunkIdx < this.chunks.length; chunkIdx++) {
-		for(var fileIdx = 0; fileIdx < this.chunks[chunkIdx].files.length; fileIdx++) {
-			if(files.indexOf(this.chunks[chunkIdx].files[fileIdx]) === -1) {
-				files.push(this.chunks[chunkIdx].files[fileIdx]);
+		for(let chunkIdx = 0; chunkIdx < this.chunks.length; chunkIdx++) {
+			for(let fileIdx = 0; fileIdx < this.chunks[chunkIdx].files.length; fileIdx++) {
+				if(files.indexOf(this.chunks[chunkIdx].files[fileIdx]) === -1) {
```

Comment:
Sorry, `Array.prototype.includes` could not use at v4.7. 
`String.prototype.includes` can be used at 4.7.